client/my-sites/google-my-business/stats/chart.js

```diff
@@ -196,11 +212,12 @@ class GoogleMyBusinessStatsChart extends Component {
 		return (
 			<Fragment>
 				<div className="gmb-stats__line-chart">
-					<LineChart
+					<LineChartWithTooltip
```

Comment:
I know that you and Yury had a discussion about the benefits of this pattern somewhere, but I cannot find it at the moment 😐

Reply:
So, removing the `renderTooltipForDatanum` from `mapStateToProps` prevents `GoogleMyBusinessStatsChart#render()` from being called each time `mapStateToProps` is called and no parameter is changed.
Using an Higher Order Component instead reduce the amount of time this props is created because we're moving it to the render function.
So yes it will force a re-render on child components even when no props for those components have changed (except for `renderTooltipForDatanum`) but we can assume that it won't happen a lot since it needs `GoogleMyBusinessStatsChart`'s props to change for that, and most of those are passed as props to the charts.

Reply:
👍